dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/shared/GridHeadersTest.java

```diff
@@ -68,7 +67,7 @@ void testFrom()
     void testFromWhenColumnsIsNull()
     {
         // Given
-        final List<Column> columns = null;
+        final Set<Column> columns = null;
```

Comment:
value columns is assigned to null and never changed and is null. Remove it and use () -> GridHeaders.from( null ) ), if it is ok for you